darts/timeseries.py

```diff
@@ -3798,11 +3798,10 @@ def plot(
             if custom_labels:
                 label_to_use = label[i]
             else:
-                label_to_use = (
-                    (label + ("_" + str(i) if len(self.components) > 1 else ""))
-                    if label != ""
-                    else "" + str(comp_name)
-                )
+                if label == "":
+                    label_to_use = comp_name
+                else:
+                    label_to_use = f"{label}_{comp_name}"
```

Comment:
I agree with adding `label` as a prefix when there are multiple components.
For a single component, I think it's better to overwrite the name with `label`. WDYT?
```suggestion
                    label_to_use = label if len(self.components) == 1 else f"{label}_{comp_name}"
```

Reply:
Fair enough. I've added it and changed the docstring accordingly.

Reply:
👍